api/gloo/enterprise.gloo/v1/auth_config.proto

```diff
@@ -302,6 +302,13 @@ message RedisOptions {
     // size of the connection pool. can leave unset for default.
     // defaults to 10 connections per every CPU
     int32 pool_size = 3;
+	// enabled with a socket type of TLS. this is the tls cert mount path for this particular host.
```

Comment:
what is going on with the indentation here?

Reply:
not sure